sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/models/FetchMetadataResult.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.iot.modelsrepository.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * The FetchMetadataResult class is used for storing the result of the
+ * fetch repository metadata operation. It contains the metadata definition
+ * and path
+ */
+@Fluent
+
+public class FetchMetadataResult {
+
+    private ModelsRepositoryMetadata definition;
+    private String path;
+    private final ObjectMapper mapper;
+
+    public FetchMetadataResult() {
+        mapper = new ObjectMapper();
```

Comment:
`ObjectMapper` should be made into a constant. Constructing them is a non-trivial operation.

Reply:
Addressed by making it static final